### PR TITLE
feat: add withdrawl instructions

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

**Review by benjamin852 — APPROVED** [review]

File: src/content/docs/validator/amplifier/verifier-rewards.mdx [hunk 1]
```diff
@@ -182,3 +182,11 @@ axelard tx wasm execute $REWARDS_CONTRACT_ADDRESS \
 Rewards are distributed to verifiers with a participation rate greater than the threshold as the epoch closes. Rewards are calculated based on the time passed since the last event and the total number of verifiers, and split equally amongst all participating verifiers in the epoch.
 
 `epoch_count` is the maximum number of historical epochs for which to distribute rewards, starting with the oldest. If not specified, rewards will be distributed for the last 10 epochs.
+
+
+## Withdrawing rewards
+By default your verifier rewards will be sent to the address of your verifier (change this with `ampd set-rewards-proxy`). To withdraw your rewards, use `ampd`'s `send-tokens` feature.
+
+```bash 
+ampd send-tokens <destination address> <amount with 6 decimals> uaxl
```
Comment (aditya-manit) [inline src/content/docs/validator/amplifier/verifier-rewards.mdx:191]: This command looks for the config in its default place, and doesnt support config flag passed to it

Comment (aditya-manit) [reply src/content/docs/validator/amplifier/verifier-rewards.mdx:191]: So if the config is in a different place, it doesnt work

Comment (aditya-manit) [reply src/content/docs/validator/amplifier/verifier-rewards.mdx:191]: Also can we make such command ask for the keyring password

Comment (aditya-manit) [reply src/content/docs/validator/amplifier/verifier-rewards.mdx:191]: @StephenFluin please take a look when you get a chance

Comment (StephenFluin) [reply src/content/docs/validator/amplifier/verifier-rewards.mdx:191]: Cc @benjamin852

**Pull request merged**